build/automation/var/profile/dev.mk

```diff
@@ -80,10 +80,10 @@ COGNITO_USER_POOL_CLIENT_ID := $(or $(COGNITO_USER_POOL_CLIENT_ID), )
 COGNITO_USER_POOL_ID := $(or $(COGNITO_USER_POOL_ID), )
 ADD_DEFAULT_COGNITO_USERS := false
 
-POSTCODE_MAPPING_SERVICE_URL := https://uec-dos-api-pc-dev-uec-dos-api-pc-ingress.k8s-nonprod.texasplatform.uk/api
+POSTCODE_MAPPING_SERVICE_URL := https://uec-dos-api-pca-dev-uec-dos-api-pc-ingress.k8s-nonprod.texasplatform.uk/api
 
 #Authentication login endpoint is set for fuzzy search at the moment. This should be configured to point authentication service api
-AUTH_LOGIN_URL := https://uec-dos-api-sfs-dev-uec-dos-api-sfs-service.k8s-nonprod.texasplatform.uk
+AUTH_LOGIN_URL := https://uec-dos-api-sfsa-dev-uec-dos-api-sfs-service.k8s-nonprod.texasplatform.uk
```

Comment:
please update the auth url in the `test/contract/environments/non-prod-deploy.postman_environment.json`

Reply:
the contract test has the correct non prod url